### PR TITLE
Carousel

### PR DESCRIPTION
**Pull request by ericletsgo**

### CODE REVIEW

File: client/app.jsx [hunk 1]
```diff
@@ -1,40 +1,107 @@
 import React from 'react';
 import axios from 'axios';
+import styled from 'styled-components';
```
Comment (John-Michael-Long) [inline client/app.jsx]: delete this

File: client/app.jsx [hunk 2]
```diff
@@ -1,40 +1,107 @@
...
+    return itemIndex - position
+  }
+
+  changeImage(event) {
+    this.setState({mainImageURL: event})
+  }
+
+  showSlides(n) {
+    var slides = document.getElementsByClassName("image-holder");
+    console.log(slides)
```
Comment (John-Michael-Long) [inline client/app.jsx]: remove console.logs

File: client/app.jsx [hunk 3]
```diff
@@ -1,40 +1,107 @@
...
+    if (n > slides.length) {
+      this.setState({slideIndex: 1});
+    }
+    if (n < 1) {
+      this.setState({slideIndex: slides.length});
+    }
+    for (var i = 0; i < slides.length; i++) {
+      slides[i].style.display = "none";
+    }
+    slides[this.state.slideIndex-1].style.display = "block";
```
Comment (John-Michael-Long) [inline client/app.jsx]: add padding between operator

File: client/app.jsx [hunk 4]
```diff
@@ -49,36 +116,35 @@ class App extends React.Component {
...
               </div>
-              <div id="description" class="flex-item">
+            </div>
+            <div className="flex-item description">
               <GameDescription description={this.state.description} descriptionImage={this.state.descriptionImage}
-              tags={this.state.tags} logoURL={this.state.logoURL} recentReviews={this.state.recentReviews}
-              allReviews={this.state.allReviews} releaseDate={this.state.allReviews} developer={this.state.developer}
-              publisher={this.state.publisher}/>
-              </div>
+                tags={this.state.tags} logoURL={this.state.logoURL} recentReviews={this.state.recentReviews}
```
Comment (John-Michael-Long) [inline client/app.jsx]: may want to put one argument per line

File: client/components/Carousel.jsx [hunk 5]
```diff
@@ -1,49 +1,11 @@
...
-                  <img src={image}  class="image"/>
-                </div>
-              )
-            }
-          )
-        }
-      </Slider>
-    )
-  }
+const CarouselSlot = (props) => {
```
Comment (John-Michael-Long) [inline client/components/Carousel.jsx]: airbnb dictates no parens around single argument

File: public/style.css [hunk 6]
```diff
@@ -1,51 +1,253 @@
...
+.links {
+  color: #66C0F4;
+}
+
+.logo {
+  float: top;
 }
 
-#description {
+.review:hover {
```
Comment (John-Michael-Long) [inline public/style.css]: remove this?

**Review by John-Michael-Long — COMMENTED** [review]
looks good, just a few comments

File: client/app.jsx (hunk 1 above)
Comment (ericletsgo) [reply client/app.jsx]: Done

File: client/app.jsx (hunk 2 above)
Comment (ericletsgo) [reply client/app.jsx]: Done

File: client/app.jsx (hunk 4 above)
Comment (ericletsgo) [reply client/app.jsx]: Done

File: client/components/Carousel.jsx (hunk 5 above)
Comment (ericletsgo) [reply client/components/Carousel.jsx]: Will fix all of this

File: public/style.css (hunk 6 above)
Comment (ericletsgo) [reply public/style.css]: Done

**Pull request merged**